File: _persist/temp/venda/CSharp/Regrafiscal.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("REGRAFISCAL")]
    public class Regrafiscal : CollectionItem
    {
        [Campo("ID_REGRAFISCAL", CampoTipo.tfKey)]
        public int Id_Regrafiscal { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_REGRAFISCAL", CampoTipo.tfReq)]
        public string Ds_Regrafiscal { get; set; }
        [Campo("IN_CALCIMPOSTO", CampoTipo.tfReq)]
        public string In_Calcimposto { get; set; }
    }
}

File: _persist/temp/venda/Java/Regrafiscal.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("REGRAFISCAL")
public class Regrafiscal extends CollectionItem
{
    @Campo("ID_REGRAFISCAL", CampoTipo.tfKey)
    public Integer Id_Regrafiscal;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("DS_REGRAFISCAL", CampoTipo.tfReq)
    public String Ds_Regrafiscal;
    @Campo("IN_CALCIMPOSTO", CampoTipo.tfReq)
    public String In_Calcimposto;
}